build/npm-scripts/ie-convert-all.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const fs = require('fs');
+const { getStylesheetPaths, transform } = require('./ie-conversion-utils');
+
+const pfStylesheetsGlob = path.resolve(__dirname, '../../dist/{components,layouts,utilities}/**/*.css');
+const patternflyBasePath = path.resolve(__dirname, '../../dist/patternfly-base.css');
+const outPath = path.resolve(__dirname, '../../dist/patternfly-ie11.css');
+
+getStylesheetPaths(pfStylesheetsGlob, [], [])
+  .then(files => {
+    if (files.length === 0) {
+      throw new Error(`getStylesheetPaths didn't receive any stylesheets to parse`);
+    }
+    return files;
+  })
+  .then(files =>
+    files.reduce((acc, file) => {
+      acc += `${fs.readFileSync(file, 'utf8')}\n`;
+      return acc;
+    }, '')
+  )
+  .then(concatCss => transform(concatCss, patternflyBasePath))
+  .then(ie11ReadyStylesheet => {
+    fs.writeFileSync(outPath, ie11ReadyStylesheet);
+    return ie11ReadyStylesheet;
+  })
+  .then(transformedCss => {
+    transformedCss.split('\n').forEach((line, index) => {
+      if (line.indexOf('undefined') >= 0) {
+        throw new Error(`Stylesheet contains undefined at line ${index + 1}`);
```

Comment:
Looking good! One last request, to help people find the problematic file faster, could we include something to point users directly to the file? As is, they only know the line number, and need to figure out what file it's in. You could add something like the following just before throwing the error.

```js
console.log(
  `\x1b[31m%s\x1b[0m`,
  `Problem in ${outPath}\n`
);
```

We use these log colors also in the a11y audit, maybe it makes sense to make these shared colors available in a better place than the a11y test config?

Then, users are guided directly to the problem.
<img width="841" alt="Screen Shot 2019-06-05 at 10 39 40 AM" src="https://user-images.githubusercontent.com/5942899/58965445-9fe93280-877e-11e9-812c-a89250acdd46.png">

Reply:
The [Chalk](https://www.npmjs.com/package/chalk) package exists for the colors, but I just used the hardcoded value for the `console.error`.